modelling/src/main/java/org/axonframework/modelling/repository/ManagedEntity.java

```diff
@@ -49,4 +49,5 @@ public interface ManagedEntity<ID, E> {
      * @return The state of the entity after the change.
      */
     E applyStateChange(UnaryOperator<E> change);
+
 }
```

Comment:
I don't think this change was necessary.
```suggestion
    E applyStateChange(UnaryOperator<E> change);
}
```